crates/red_knot_python_semantic/resources/mdtest/annotations/unsupported_special_forms.md

```diff
@@ -91,7 +91,9 @@ Some of these are not subscriptable:
 ```py
 from typing_extensions import Self, TypeAlias
 
-X: TypeAlias[T] = int  # error: [invalid-type-form]
+# error: [invalid-type-form] "Special form `typing.TypeAlias` expected no type parameter"
+# error: [unresolved-reference] "Name `T` used when not defined"
```

Comment:
I think we just forgot to define a `TypeVar` here 😆 can we add `T = TypeVar("T")` immediately above this to get rid of the `unresolved-reference` diagnostic? I think it distracts from what this is meant to be testing

Reply:
Of course. I thought it should be a type variable, but wasn't sure if that was intended to be implicitly declared…